arm7/lib/include/OS_timer.h

```diff
@@ -0,0 +1,32 @@
+#ifndef POKEDIAMOND_OS_TIMER_H
+#define POKEDIAMOND_OS_TIMER_H
+
+#include "consts.h"
+
+typedef enum
```

Comment:
I think that these can be in shared? can't check right now, but please check if the values are shared between 7 and 9 and if so move them there so to avoid code duplication